seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -61,13 +61,23 @@ angular.module('BE.seed.controller.menu', [])
       });
       $scope.$on('app_success', function () {
         $rootScope.route_load_error = false;
-      })
+      });
       $scope.$on('organization_list_updated', function () {
         init();
       });
 
       $scope.is_active = function (menu_item) {
-        if (menu_item === $location.path()) {
+        if ($state.current.url !== '^') {
+          $location.search($location.path(), $state.current.url);
+        }
+        if($rootScope.stay_on_page && menu_item === '/' + $state.current.url.split('/')[1]) {
+          return true;
+        } else if (menu_item === $location.path()) {
+          if ($rootScope.stay_on_page) {
+            return false;
+          } else if (!$rootScope.stay_on_page && menu_item === ('/' + $state.current.url.split('/')[1])) {
+            return true;
```

Comment:
Cases where require resetting active tabs (changed from light gray to dark gray).